scripts/scrape-fenix

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+"""
+Toolkit for scraping GitHub feature labels from Fenix data reviews and using
+them to create a preliminary set of metric annotations.
+
+Usage:
+
+./scripts/update_metric_labels
+
+This is intended as a stopgap until we can incorporate this information directly
+into Fenix itself. See:
+
+https://docs.google.com/document/d/1K-Hl_ZWRvVzqBAGzdXb_oQY52D-D5YDxorYmzyzfxuw/edit#
+"""
```

Comment:
I consolidated all the miscellaneous scraper logic into this single script, which will hopefully be easier to reason about.